src/Data/Container/GreatestFixpoint.agda

```diff
@@ -0,0 +1,23 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Greatest fixpoint for containers
+------------------------------------------------------------------------
+
+{-# OPTIONS --safe --without-K --guardedness #-}
+
+module Data.Container.GreatestFixpoint where
```

Comment:
To have a consistent naming convention with the sized variant, should this go under `Data.Container.Fixpoint.Guarded`? (and then publicly re-export `\mu` from `Data.Container`?)

Reply:
I like this.